### DISCUSSION TITLE
Server Error Error: supabaseUrl is required.

### DISCUSSION THREAD

**Discussion by MarcJamesIO** [supabase/supabase#6459] [Questions]
Hi I've followed the NextJS starter article and I'm getting 
"Server Error: supabaseUrl is required."

My .env.local is in my root with credentials as per the article https://supabase.com/docs/guides/with-nextjs

And the error points to the last line of your script here:

`import { createClient } from '@supabase/supabase-js'

const supabaseUrl = process.env.NEXT_PUBLIC_SUPABASE_URL
const supabaseAnonKey = process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY

export const supabase = createClient(supabaseUrl, supabaseAnonKey)
`

Has anyone else experienced this?

**Comment by GaryAustin1** [comment]
This has some NextJS comments dealing with setup of .env file and development environment.
https://github.com/supabase/supabase/discussions/3218
Basically the error is your supabaseURL variable is not getting set.

**Reply by Prudence97** [reply]
Did you figure this out. I'm having the same problem 🙇‍♀️
